go.mod

```diff
@@ -0,0 +1,25 @@
+module github.com/palantir/witchcraft-go-server
+
+go 1.13
+
+require (
+	github.com/gorilla/mux v1.7.3
+	github.com/julienschmidt/httprouter v1.3.0
+	github.com/nmiyake/pkg/dirs v1.0.0
+	github.com/palantir/go-encrypted-config-value v1.1.0
+	github.com/palantir/pkg/httpserver v1.0.0
+	github.com/palantir/pkg/metrics v1.0.0
+	github.com/palantir/pkg/objmatcher v1.0.0
+	github.com/palantir/pkg/safejson v1.0.0
+	github.com/palantir/pkg/safeyaml v1.0.0
+	github.com/palantir/pkg/signals v1.0.0
+	github.com/palantir/pkg/tlsconfig v1.0.0
+	github.com/palantir/witchcraft-go-error v1.2.0
+	github.com/palantir/witchcraft-go-logging v1.5.0
+	github.com/palantir/witchcraft-go-params v1.1.0
+	github.com/palantir/witchcraft-go-tracing v1.2.0
+	github.com/stretchr/testify v1.4.0
+	gopkg.in/fsnotify.v1 v1.4.7
```

Comment:
Should we apply a `replace` directive for this import to `github.com/fsnotify/fsnotify`? We could alternatively import it from `github.com` directly rather than `gopkg.in` here:
https://github.com/palantir/witchcraft-go-server/blob/develop/witchcraft/refreshable/refreshable_file.go#L27

Reply:
Since this is meant to be a library, would like to avoid replace directives (since they're not inherited). Fine with moving this import over to `fsnotify/fsnotify` as long as we don't have other dependencies of this project that pull in `gopkg.in/fsnotify`. Not sure if one is considered more canonical over another -- for public .com repos, the "gopkg.in" path is imported by 1,201 packages (https://godoc.org/gopkg.in/fsnotify.v1) and and "fsnotify/fsnotify" is imported by 1,352 packages (https://godoc.org/github.com/fsnotify/fsnotify), so it seems to be fairly close. Couldn't see a definitive split internally.

It looks like "fsnotify/fsnotify" may be the more canonical path, so fine with switching with that. However, will note that this is not universally true -- for example, for `gopkg.in/yaml.v2`, the "gopkg.in" path is considered canonical, even though the repo itself is also in github.com.

Reply:
That makes sense. I was more just curious if that would be a better approach, but also didn't find consensus on what the best practice was here, which seems like there really isn't one currently. 
The reason I mentioned this, was because I noticed in other repositories that vendor this I was getting the following error when those repositories have the replace directive, and thought this could be an issue for consumers.

```
go: github.com/fsnotify/fsnotify@v1.4.7 used for two different module paths (github.com/fsnotify/fsnotify and gopkg.in/fsnotify.v1)
```

The downside to changing it to using the `github.com` is that now consumers will need the replace directive to get around the above issues.